### ISSUE TITLE
Use Administrator Privilege to install c4d plugin properly

### ISSUE THREAD

**Issue by eZioPan** [NiklasRosenstein/c4ddev#17]
With release v0.1.6, if we copy C4DDev plugin into c4d plugin directory, we need start c4d with administrator privilege for the first time, to let C4DDev plugin initialize itself. After that we can use c4d and plugin normally.

If we don't use Administrator privilege, then we will get IOError in C4D Script Console

Could this information to https://niklasrosenstein.github.io/c4ddev/getting-started/ to avoid installation problem?

**Comment by NiklasRosenstein** [comment]
Hi @eZioPan, thanks for raising this issue! Actually, this is not intended and probably a bug in [Node.py](https://github.com/nodepy/nodepy). While it will optimize the Python source code to bytecache files on the first run when under an Administrator account, it should not result in an error when it doesn't have write privileges (which I assume is the reason for the error?).

I'll try to reproduce this issue here. I haven't come accross it because I installed C4D on my User's home directory.

**Comment by eZioPan** [comment]
Yes, I think Node.py causes it. But for most situation, Node.py just does right. Since most python source code will be place in directory that user has privilege to write. But as you know, the default installation directory of C4D is "C:\Program File\MAXON\", which exactly need Administrator privilege to write. So just for now, it's better let user know how to fix it.
By the way, since user always has write permission of "C:\Users\<UserName>\AppData\Roaming\MAXON", maybe place plugin in this directory is an other workaround.
